spec/mailers/application_mailer_spec.rb

```diff
@@ -0,0 +1,12 @@
+class UserMailer < ApplicationMailer
+  include ActiveJob::TestHelper
+  def test_email; end
+end
```

Comment:
Just thinking allow, what if we put this in the `support` folder just to show how a mock could be done? I usually prefer them close to the code too so doubtful about it

Reply:
I would prefer this to be deleted after some real tests.... so I prefer to keep it here and developers deleting it later, sounds good?